static/index.html

```diff
@@ -110,12 +120,10 @@ <h2 class="subheader">Together, we can make a difference in just 10 minutes a we
     <div class="row">
         <div class="col-md-7">
             <div class="intro">
-                <p class="lg">It's easier than you think to do something to advance women and equality.</p>
-                <p>Calling members of Congress is one of the <a href="https://www.nytimes.com/2016/11/22/us/politics/heres-why-you-should-call-not-email-your-legislators.html">most effective ways</a> to influence the political process, and Calls for Change makes it <a href="#how-it-works">simple and fast</a>. Each week we'll send you all the information you need &mdash; from the right phone number to a sample script &mdash; so in just ten minutes you can make sure your voice is heard. Learn more about the <a href="#about">issues we’re focused on</a>.</p>
+                <p><strong>It's easier than you think to do something to advance women and equality.</strong> Calling members of Congress is one of the <a href="https://www.nytimes.com/2016/11/22/us/politics/heres-why-you-should-call-not-email-your-legislators.html">most effective ways</a> to influence the political process, and Calls for Change makes it <a href="#how-it-works">simple and fast</a>. Each week we'll send you all the information you need &mdash; from the right phone number to a sample script &mdash; so in just ten minutes you can make sure your voice is heard. <a href="#about">Learn more about the issues we’re focused on</a>.</p>
```

Comment:
This looks like a design revert back to what's currently in https://www.callsforchange.com — isn't https://www.52calls-52weeks.com (our staging environment) more up-to-date?